packages/mobile/src/exchange/ExchangeHomeScreen.tsx

```diff
@@ -1,20 +1,18 @@
+import Button, { BtnTypes } from '@celo/react-components/components/Button'
 import ScrollContainer from '@celo/react-components/components/ScrollContainer'
 import SectionHeadNew from '@celo/react-components/components/SectionHeadNew'
-import SmallButton from '@celo/react-components/components/SmallButton'
 import BigNumber from 'bignumber.js'
 import * as React from 'react'
 import { withNamespaces, WithNamespaces } from 'react-i18next'
 import { StyleSheet, View } from 'react-native'
 import SafeAreaView from 'react-native-safe-area-view'
 import { connect } from 'react-redux'
-import CeloAnalytics from 'src/analytics/CeloAnalytics'
-import { CustomEventNames } from 'src/analytics/constants'
 import componentWithAnalytics from 'src/analytics/wrapper'
 import AccountOverview from 'src/components/AccountOverview'
 import { fetchExchangeRate } from 'src/exchange/actions'
 import Activity from 'src/exchange/Activity'
 import ExchangeRate from 'src/exchange/ExchangeRate'
-import { CURRENCY_ENUM as Token } from 'src/geth/consts'
```

Comment:
We now use `CURRENCY_ENUM` everywhere in the app.. no more ` CURRENCY_ENUM as Token` silliness